### ISSUE TITLE
References id's

### ISSUE THREAD

**Issue by jlgelpi** [inab/benchmarking-data-model#10]
Using DOI's  as primary id generates a problem with REST API as then usually contain '/' characters. They should be escaped or replaced usign some criteria.

**Comment by jmfernandez** [comment]
I agree with you. The 'slash' character should be represented using its code %2F , for instance (i.e. urlencoding the parameters)

**Comment by jlgelpi** [comment]
Some references do not have DOI (no idea why) and others are not papers (PDB reference entries in CAMEO). 
Could think in an abstract way of solving using prefixed ids: 
DOI:...
PUBMED:...
PDB:...
Potential problem: Duplicated papers.

**Comment by jmfernandez** [comment]
Yes, that's true. Perhaps in the mid/long term we need some reference canonicalization step. In the literature case, it should be based on DOI, as all the PubMed Ids should have a DOI, but not all the DOIs have a PubMed id. We should rely on existing services, like identifiers.org . But I do not know how well it behaves on outdated identifiers (for instance, accession numbers from UniProt).

**Comment by JuergenHaasSIB** [comment]
PDB registers DOIs as well: 10.2210/pdb2ndk/pdb, so we can specify them.

**Comment by JuergenHaasSIB** [comment]
PubMed API does not consistently resolve DOIs - we use it in Modelarchive.org to obtain references. They are looking into it.

**Comment by jlgelpi** [comment]
Will move PDB references to DOI to make them consistent

**Comment by diglesia** [comment]
What could we use for Pubmed references without a DOI? I have some cases in QfO. I was able to find the DOI for some of them searching in Google directly but not for all the cases. For such cases, I have used (provisionally) the URI to the PubMed record.
With regards to PDB references, what would we use as "title" (required field in Reference): the short name (e.g. 5GTP) or the long one (e.g.The agonist-free structure of human PPARgamma ligand binding domain in the presence of the SRC-1 coactivator peptide)?
